frontend/src/metabase/user/components/UserSettings.jsx

```diff
@@ -12,13 +12,37 @@ import UserAvatar from "metabase/components/UserAvatar";
 import SetUserPassword from "./SetUserPassword";
 
 export default class UserSettings extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { updateUserResult: null };
```

Comment:
If you do the change `submitResultDetail` to `formError`/`formSuccess` props, I'd just have 

```
this.state = {
  formError: null,
  formSuccess: null
};
```